service/scaladsl/api/src/main/scala/com/lightbend/lagom/scaladsl/api/deser/PathParamSerializer.scala

```diff
@@ -99,4 +101,25 @@ object PathParamSerializer {
       }
     }
   }
+
+  /**
+   * A traversable path param serializer
+   */
+  implicit def traversablePathParamSerializer[CC[X] <: Traversable[X], Param: PathParamSerializer](implicit bf: CanBuildFrom[CC[_], Param, CC[Param]]): PathParamSerializer[CC[Param]] = {
+    val delegate = implicitly[PathParamSerializer[Param]]
+    val name = delegate match {
+      case named: NamedPathParamSerializer[Param] => s"Traversable[${named.name}]"
```

Comment:
@sirocchj we may want to remove the type param on this `case`. 

`case named: NamedPathParamSerializer[_]` is better. 

The type param generates and compiler warning because of type erasure and it's not relevant to generate the name.

Reply:
yep, noticed it on option too, agree it should be removed